website/index.html

```diff
@@ -21,87 +21,94 @@ <h2>
             <div class="line">
                 <div></div>
             </div>
-            <div id="main-content">
+            <div class="main-content">
                 <div class="text-image-flex">
                     <div class="introduction width-mobile">
                         <p>
                             <span>Hi! I am a developer</span>
-                            <br>
-                            <br>
+                            <br/>
+                            <br/>
                             I am a front end and 
-                            back end developer, that creates technical solutions for <br>
+                            back end developer, that creates technical solutions for <br/>
                             companies/clients of all genres.
-                            <br>
-                            <br>
+                            <br/>
+                            <br/>
                             Want to see some of my work? 
-                            <br>
+                            <br/>
                             ...You are looking at it!
                         </p>
                     </div>
                     <div class="image-container-3">
-                        <img src="image-1.png" alt="image could not be displayed">
+                        <img src="static/images/portrate.png" alt="Image could not be displayed"/>
                     </div>
                 </div>
                 <div class="text-image-flex">
                     <div class="development-for-multible-mediums width-mobile">
                         <p>
-                            <span>Development for multible mediums:</span><br>
-                            <br>
+                            <span>Development for multible mediums:</span>
+                            <br/>
+                            <br/>
                             <ul>
                                 <li>Web application development,</li>
                                 <li>Smartphone app development. </li>
                             </ul>
-                            <br>  
+                            <br/>  
                         </p>
                         <p>
-                            <span>Types of development:</span><br>
-                            <br>
+                            <span>Types of development:</span>
+                            <br/>
+                            <br/>
                             <ul>
                                 <li>Frontend & graphical user interface,</li>
                                 <li>Backend & server development.</li>
                             </ul>
                         </p>  
                     </div>
                     <div class="image-container-1">
-                        <img src="image-2.png" alt="image could not be displayed">
+                        <img src="static/images/different-mediums.png" alt="Image could not be displayed"/>
```

Comment:
Incorrect use of alt tag: [alt text](https://moz.com/learn/seo/alt-text)

Reply:
51ce2a4